src/hooks/useQuizState/useQuizApiState.ts

```diff
@@ -75,6 +90,13 @@ const useQuizApiState: UseQuizApiState = (
     }
   };
 
+  useEffect(() => {
```

Comment:
I think this fixed the double fetch issue but introduced a new one. If you go through the quiz once, everything works fine. If you click on retake and take it again, the config is not fetched and on top of that the even the default title doesn't render.
![Screenshot 2023-12-01 at 9 51 24 AM](https://github.com/Constructor-io/constructorio-ui-quizzes/assets/2954126/406f302e-4eaa-4bd1-91f8-76a2602d71e6)

Reply:
Fixed here [17dee0c](https://github.com/Constructor-io/constructorio-ui-quizzes/pull/155/commits/17dee0c0f773c9e17295bc829bdca75424594341)